boutiques_descriptors/cellranger_count_7.0.1.json

```diff
@@ -13,22 +13,23 @@
             "type": "File",
             "name": "FASTQs data",
             "id": "fastqs",
-            "description": "Input FASTQ data",
+            "description": "Input FASTQ data.\n Note: You cannot use a Cbcsv file to launch multiple tasks. It will conflict with the --sample option.",
```

Comment:
I don't like that the descriptor mentions CBRAIN-specific concepts, it should always be a general description of the tool, usable outside of CBRAIN.